src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp

```diff
@@ -957,7 +990,11 @@ void EditModeConstraintCoinManager::processConstraints(const GeoListFacade& geol
                         dir.RotateZ(-M_PI / 2.0);
 
                         relPos = seekConstraintPosition(
-                            pos, norm, dir, 2.5, editModeScenegraphNodes.constrGroup->getChild(i));

```

Comment:
This is IMO not improving the syntax. I would rather see this as a one liner and here it takes 6 lines.